### DISCUSSION TITLE
Total count in Friends leaderboard

### DISCUSSION THREAD

**Discussion by M4N0JKUM4R** [cssbattle/forum#90] [Bugs reports]
![image](https://user-images.githubusercontent.com/11471878/130331550-543d435a-2354-4f8a-af3b-3d559dae4ce3.png)

I am currently ranked #1 among the 122k+ users when I have the friends option selected. Shouldn't the total count be 2? My current global rank is 108.

**Comment by chinchang** [comment]
fixed now. Thanks for reporting :)
